### PR TITLE
ELIoT

### PR DESCRIPTION
**Pull request by bluelancer**
A simulation of LWM2M communication between Digital twins and LWM2M server, In both source code and docker image.

Reference: https://github.com/Alliasd/ELIoT/wiki/Emulated-IoT-(ELIoT)-Platform

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@
 *.bak
 *.swp
 *~.nib
-*~
\ No newline at end of file
+*~
+ELIoT/CoapNode/src/app/node_modules/
```
Comment (berezovskyi) [inline .gitignore]: would be best to have a separate `.gitignore` file in `ELIoT/CoapNode/`

File: ELIoT/CoapNode/src/.idea/modules.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (berezovskyi) [inline ELIoT/CoapNode/src/.idea/modules.xml]: I think IntelliJ folder should not be there.

File: ELIoT/demo/ELIoT/devices/out/Refresh_coordinate.html [hunk 3]
```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
```
Comment (berezovskyi) [inline ELIoT/demo/ELIoT/devices/out/Refresh_coordinate.html]: Shall the `/out/` folder be ignored?

File: ELIoT/demo/ELIoT/jsdoc-toolkit/app/frame/Dumper.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
+/**
```
Comment (berezovskyi) [inline ELIoT/demo/ELIoT/jsdoc-toolkit/app/frame/Dumper.js]: Do we need to share the `jsdoc-toolkit` code here as well?

File: ELIoT/leshan/leshan.iml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (berezovskyi) [inline ELIoT/leshan/leshan.iml]: We need to remove IntelliJ and JAR files from here.

File: mqtt/README.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+
+Purpose: MQTT broker
```
Comment (berezovskyi) [inline mqtt/README.md]: What is this?

**Review by berezovskyi — CHANGES_REQUESTED** [review]

**Comment by bluelancer** [comment]
@berezovskyi maybe better now?

**Review by berezovskyi — APPROVED** [review]

**Pull request merged**